src/main/java/com/puppycrawl/tools/checkstyle/checks/coding/FinalLocalVariableCheck.java

```diff
@@ -141,6 +155,18 @@ public final void setValidateEnhancedForLoopVariable(boolean validateEnhancedFor
         this.validateEnhancedForLoopVariable = validateEnhancedForLoopVariable;
     }
 
+    /**
+     * Setter to control whether to check
+     * <a href="https://docs.oracle.com/javase/specs/jls/se21/preview/specs/unnamed-jls.html">
+     * unnamed variables</a>.
+     *
+     * @param validateUnnamedVariables whether to check unnamed variables
+     * @since 10.18
```

Comment:
@nrmancuso @romani 
23 matches of `X.X.X`. Most seem to be `TokenTypes` and `SuppressWithNearbyTextFilter `. 529 matches of `X.X`. Shouldn't we always be using 3 digit versions everywhere now?

Reply:
Yes, we should be. Let's create an issue on this, maybe we can figure out some way to enforce this.

@mahfouz72 please update this to be Major.Minor.Patch

Reply:
done, https://github.com/checkstyle/checkstyle/issues/14995

Reply:
I closed this you created one before 😄 https://github.com/checkstyle/checkstyle/issues/12653

Reply:
I completely forgot about this :)

Reply:
if we stapiing in this problem again and again, 
we can enforce 3 digits and have long list of allowed 2 digit version, long list is ok, it was not done only because we thought we will unlikley do this mistake again :).
by means of few greps on source code you can get a list relatively quikly

Reply:
Resolved.